src/client/src/app/components/events/event-timeslot/event-timeslot.component.html

```diff
@@ -112,7 +112,7 @@ <h2 class="m-0 grow">
                     <span class="flex w-full flex-row items-center gap-2">
                       <span class="min-w-0 grow truncate">{{
                         translations.events_playerAmount()
-                          | interpolate: { amount: preconfig.playerIds.length }
+                          | interpolate: { amount: preconfig.playerIds.length.toString() }
```

Comment:
Does this not throw an error? As length is undefined you should not be able to call toString on it. You can do `.length || 0` instead.

Reply:
Actually when directly typing 0 as an int, it will be displayed as "undefined". Maybe then something like 
.length.toString() || "0"

Reply:
I don't think that the preconfigs.playerIds array ever can get undefined as it is defined as 
``` ts
export type ApiEventInstancePreconfiguration = {
    id: string;
    playerIds: (string)[];
  };
```

Reply:
This was a bug in the @ngneers/signal-translate package and has been fixed in 0.1.1.